Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -3792,106 +3791,79 @@ internal async static Task ObjectRetentionAsync_Test1(MinioClient minio)
 
         #endregion
 
-        #region Bucket Replication
-        internal async static Task BucketReplicationAsync_Test1(MinioClient minio, string remoteARN)
+
+        #region Bucket Lifecycle
+        internal async static Task BucketLifecycleAsync_Test1(MinioClient minio)
         {
             DateTime startTime = DateTime.Now;
-            string bucketName1 = GetRandomName(15);
-            string bucketName2 = GetRandomName(15);
-            string objectName = GetRandomObjectName(10);
+            string bucketName = GetRandomName(15);
             var args = new Dictionary<string, string>
             {
-                { "bucketName1", bucketName1 },
-                { "bucketName2", bucketName2 }
+                { "bucketName", bucketName }
             };
-            ReplicationConfiguration repl;
 
+            List<LifecycleRule> rules = new List<LifecycleRule>();
+            Expiration exp = new Expiration(DateTime.Now.AddYears(1));
+            var compareDate = new DateTime(DateTime.Now.Year, DateTime.Now.Month, DateTime.Now.Day, 0 ,0 ,0);
+            var expInDays = (compareDate.AddYears(1) - compareDate).TotalDays;
+
+            LifecycleRule rule1 = new LifecycleRule(null, "txt", exp, null,
+                new RuleFilter(null, "txt/", null),
+                null, null, LifecycleRule.LIFECYCLE_RULE_STATUS_ENABLED
+                );
+            rules.Add(rule1);
+            LifecycleConfiguration lfc = new LifecycleConfiguration(rules);
```

Comment:
are these added to this PR in error?

Reply:
All changes to functional tests removed.